src/xdocs/config_imports.xml

```diff
@@ -1557,22 +1557,30 @@ import java.util.stream.IntStream;
           <li>groups are separated by, at least, one blank line and aren't separated internally</li>
         </ul>
 
-        <p>
-        Note: &quot;separated&quot; option is disabled because IDEA default has blank line
-        between &quot;java&quot; and static imports, and no blank line between
-        &quot;javax&quot; and &quot;java&quot;
-        </p>
+        <p>Note: a <a href="config_filters.html#SuppressionFilter">suppression filter</a>
+           is needed because IDEA has no blank line between "javax" and "java".</p>
```

Comment:
please add  `ImportOrder has by design limitation to enforce empty line between groups ("java", "javax"), there is no flexibility to enforce empty lines between some groups and no empty lines between another groups.`